### ISSUE TITLE
Update main page project summaries

### ISSUE THREAD

**Issue by julch94** [julch94/julch94.github.io#1]
- [x] Ultimate Prey
- [x] Chick Game
- [x] Waifu Hunter 2077
- [x] GG 2
- [x] GG
- [x] Smaller Projects

- banner image to include on main page

- Link to playable demo

- Game Summary

- Key Skills and Lessons

- Made with [Tools]

Go look at below link for existing write-ups:
https://github.com/RealShitGamers/.github

**Comment by julch94** [comment]
## Chick Game

![Chick Game Banner](/assets/chick_game_25.png)

[Gameplay and More for Chick Game - Click Here](/chick_game/)

Chick Game is an FPS where you play as a rooster who must care and tend to his hens and chicks during the day and defend the coop at night from waves of different predators. 

**Top 5 Skills and Lessons Learned:**
*   Developing useful milestones and projects using GitHub
*   Creating custom animations in both Blender and Unity using models from the Unity Asset Store
*   Unity Audio System
*   Developing design documentation for another developer to implement
*   Troubleshooting complex game logic and developing custom solutions to do so

Made with Unity, C#, Blender, and a mix of free and paid assets.

**Comment by julch94** [comment]
NOTE: Jekyll ignores folders with underscores in front of them. So had to rename _assets to assets where I stored the screenshot. Frustrating.

**Comment by julch94** [comment]
## Ultimate Prey

![Ultimate Prey Banner](/assets/ultimate_prey_25.png)

[Gameplay and More for Ultimate Prey - Click Here](/ultimate_prey/)

// Game Description here

**Top 5 Skills and Lessons Learned:**
*   State Machines and Behavior Trees
*   LOD Models
*   Multiplayer Networking
*   Developing and utilizing re-usable functions, scripts (including some utilities), and prefabs
*   Play-testing and gathering feedback

Made with Unity, C#, Audacity, Mirror Networking for Unity, and a mix of free and paid assets

**Comment by julch94** [comment]
NOTE: These comments are not the authoritative copy - see the index.markdown file for those as I'm simply using these comments to develop the rough draft for those.

**Comment by julch94** [comment]
GG2

Summary:
GG2 is a top down dungeon crawler built to run in a web browser using HTML5. You play as a mage who must explore each level, collecting new abilities and upgrades, and defeat 3 main bosses to get to the final boss. This game was my final project using the GameMaker engine. If I spent just one day to work on an upgrade to this game it would be to scrap most of the enemies from each level and fix the monitor sync rate issue. This would let the cool bosses and systems of the game really shine. 

Key skills and lessons:
- Quality over quantity. The bosses have so much more thought and love put into them compared to the rest of the dungeon rooms and it shows as those are much less interesting. 
- Platform testing and understanding hardware impacts. The game speed is sync's with monitor refresh rate which is a major issue. 
- How to use resource packs and scale/adapt them to your game's implementation.
- Level design and system design. I spent a significant portion of this project designing the flow of the overall game and interactions of each system.
- GameMaker was great to learn with but it was time to move to Unity or Unreal Engine.

**Comment by julch94** [comment]
GG

Summary
GG is my first complete video game  I made from start to finish. It's a 2D side-scroller inspired by those frustrating flash game with stupid obstacles/puzzles and expectation for you to die, learn, repeat, and eventually win. I made this game using GameMaker as a free and accessible engine and learned a lot about programming for game development using the C++ variant called GameMaker Language (GML). It was also my first experience mapping out the scope of a game including the number of levels, enemy types, objectives, mechanics, and more. I added Easter eggs and a game run timer and challenged my friends and family to be the first to find all the Easter eggs and have the fastest run by some determined date. I also asked them all for feedback on the game. I rewarded the winners with $10 Amazon cards which was a fun way to thank my friends and family for helping me improve as a game designer and developer. 

Lessons and skills
- Reignited my programming skills and gave me an opportunity to design a game. 
- Planning, designing, and developing an entire (very simple) game. 
- Using a game engine. 
- Play testing and feedback. 
- Simple is good. Fun is better.
